packages/headless/doc-parser/src/interface-resolver.ts

```diff
@@ -23,10 +23,19 @@ import {
 
 export function resolveInterfaceMembers(
   entry: ApiEntryPoint,
-  apiInterface: ApiInterface
+  apiInterface: ApiInterface,
+  ancestorNames: string[]
 ): AnyEntity[] {
-  const members = resolveMembers(entry, apiInterface);
-  const inheritedMembers = resolveInheritedMembers(entry, apiInterface);
+  if (ancestorNames.includes(apiInterface.name)) {
```

Comment:
In the JSON, under `extractedTypes`, the `CategoryFacetValue#children` member has type `CategoryFacetValue`, and `isTypeExtracted` set to `false`.

What would happen if instead of using `includes` here, we checked if the name was present two times here instead of once before returning an empty array?

Reply:
I'm not sure why we would want to parse the children an additional time 🤔

If we checked that an interface was present two times, it would show the members of an interface twice down a given tree.

Reply:
I think having one additional layer will allow the the extractor to set `isTypeExtracted` to `true` on the first repeated instance, making it possible to link in the docs.